src/login/logind-dbus.c

```diff
@@ -2040,17 +2040,23 @@ static int verify_shutdown_creds(
         }
 
         if (blocked) {
-                /* We don't check polkit for root here, because you can't be more privileged than root */
-                if (uid == 0 && (flags & SD_LOGIND_ROOT_CHECK_INHIBITORS))
```

Comment:
this check should be retained for weak inhibitors. the way i read this, this isn't respected at all anymore.

Reply:
This was added back in 845f95b9e00.